cypress/integration/Homepage.test.js

```diff
@@ -1,7 +1,7 @@
 describe("Landing Page", () => {
   beforeEach(() => {
     cy.visit("/");
-    cy.shouldHaveLargeHeading();
+    cy.shouldHaveFooter();
```

Comment:
duplicate

Reply:
Sorry I didn't notice that. Removed now.